pkg/openstack/loadbalancer.go

```diff
@@ -1310,11 +1332,14 @@ func (lbaas *LbaasV2) buildBatchUpdateMemberOpts(port corev1.ServicePort, nodes
 		}
 
 		if port.NodePort != 0 { // It's 0 when AllocateLoadBalancerNodePorts=False
+			customTag := lbaas.getCustomLoadBalancerTags(service, svcConf)
```

Comment:
I think this is pretty much constant, so we can do this just once, outside of the loop over `nodes`.